python/tvm/runtime/module.py

```diff
@@ -282,7 +293,22 @@ def export_library(self,
             self.save(file_name)
             return
 
-        modules = self._collect_dso_modules()
+        graph_runtime_factory_modules = self._collect_modules("GraphRuntimeFactory")
+        for index, module in enumerate(graph_runtime_factory_modules):
+            if not package_params:
+                module.get_function("diable_package_params")()
+                params_file_name = "deploy_" + module.get_function("get_module_name")() + ".params"
```

Comment:
We should not add any logics specific to GraphRuntimeFactory here, and directly use the original serialization mechanism. The additional behavior(save `params` if not packaging params) brings inconsistency to the API. We would rather let the user call `get_params` by themselves and save it in this case.